massa-execution-worker/src/tests/scenarios_mandatories.rs

```diff
@@ -1,3115 +1,2484 @@
-            let keypair = KeyPair::from_str(TEST_SK_1).unwrap();
-            let event_test_data = include_bytes!("./wasm/event_test.wasm");
-            let operation =
-                create_execute_sc_operation(&keypair, event_test_data, BTreeMap::default())
-                    .unwrap();
-            let blockclique_block = create_block(
-                keypair,
-                vec![operation.clone()],
-                vec![],
-                blockclique_block_slot,
-            )
-            .unwrap();
-            blockclique_blocks.insert(blockclique_block_slot, blockclique_block.id);
-            let mut blockclique_block_storage = storage.clone_without_refs();
-            blockclique_block_storage.store_block(blockclique_block.clone());
-            blockclique_block_storage.store_operations(vec![operation]);
-            block_metadata.insert(
-                blockclique_block.id,
-                ExecutionBlockMetadata {
-                    storage: Some(blockclique_block_storage),
-                    same_thread_parent_creator: Some(get_random_address()),
+                ExecutionQueryRequestItem::AddressDatastoreKeysFinal {
+                    addr,
+                    prefix: vec![],
                 },
-            );
-        }
-        // notify execution about blockclique change
-        controller.update_blockclique_status(
-            Default::default(),
-            Some(blockclique_blocks.clone()),
-            block_metadata.clone(),
-        );
-        std::thread::sleep(Duration::from_millis(1000));
-        let events = controller.get_filtered_sc_output_event(EventFilter::default());
-        assert_eq!(events.len(), 2, "wrong event count");
-        assert_eq!(events[0].context.slot, Slot::new(1, 0), "Wrong event slot");
-        assert_eq!(events[1].context.slot, Slot::new(1, 1), "Wrong event slot");
+                ExecutionQueryRequestItem::AddressDatastoreValueCandidate {
+                    addr,
+                    key: key_a.clone(),
+                },
+                ExecutionQueryRequestItem::AddressDatastoreValueFinal {
+                    addr,
+                    key: key_a.clone(),
+                },
+                ExecutionQueryRequestItem::OpExecutionStatusCandidate(operation.id),
+                ExecutionQueryRequestItem::OpExecutionStatusFinal(operation.id),
+                ExecutionQueryRequestItem::AddressRollsCandidate(addr),
+                ExecutionQueryRequestItem::AddressRollsFinal(addr),
+                ExecutionQueryRequestItem::AddressDeferredCreditsCandidate(addr),
+                ExecutionQueryRequestItem::AddressDeferredCreditsFinal(addr),
+                ExecutionQueryRequestItem::CycleInfos {
+                    cycle: 0,
+                    restrict_to_addresses: None,
+                },
+                ExecutionQueryRequestItem::Events(EventFilter::default()),
+            ],
+        });
+    universe.module_controller.get_addresses_infos(&[addr]);
```

Comment:
I don't understand why you make these calls. You don't check the values, is it just to check that the calls succeed?

Reply:
Yes. I didn't add a assert of the values, just wanted to verify that it suceed I can add asserts if you want

Reply:
No need for asserts, but a comment would be nice :)

Reply:
Done